packages/model-viewer/src/features/loading.ts

```diff
@@ -416,6 +424,10 @@ export const LoadingMixin = <T extends Constructor<ModelViewerElementBase>>(
           this[$posterContainerElement].classList.remove('quick');
         }
       }
+
+      if (changedProperties.has('withCrendentials') || this.withCredentials === true) {
```

Comment:
No need for the `||` statement; checking that it has changed is enough.

Reply:
If I do not add the `||` statement, the object `changedProperties` do not have the `withCredentials` value. Maybe I'm missing something ?

Reply:
`this` always has `withCredentials`; the `changedProperties` is a subset. Follow the similar statements above. If it's not behaving as expected, let me know.

Reply:
![image](https://user-images.githubusercontent.com/6912564/156658790-c78d0227-105d-49fe-bc1f-b67438d4522f.png)

`updated` is triggered, but the `changedProperties` subset do not have the `withCredentials` to true. It's undefined on the first trigger, and no preset in the other console.log.

Reply:
I think I found my issue !
I'm not adding that at the right place. I was editing the `loading.ts` file instead of `model-viewer-base.ts` one.

I'm pushing a new PR change.